rules/CodeQuality/Rector/ClassConstFetch/ConvertStaticPrivateConstantToSelfRector.php

```diff
@@ -43,10 +75,12 @@ public function run()
 final class Foo
 {
     private const BAR = 'bar';
+    public const BAZ = 'baz';
 
     public function run()
     {
         $bar = self::BAR;
+        $baz = self::BAZ;
     }
 }
```

Comment:
Could you update the code sample to be just 1 item? It should not mimic the test fixtures and cover all cases, but the most probable one, to make it easier to read and generate docs from.